cmd/fleetctl/apple_mdm.go

```diff
@@ -50,172 +49,6 @@ func appleMDMCommand() *cli.Command {
 	}
 }
 
-func generateCommand() *cli.Command {
```

Comment:
This was just moved to the `generate.go` file, so that each command is in its own similarly-named file. The `apple-mdm` command will be removed soon, so this whole `apple_mdm.go` file will then be deleted.